GliaWidgetsTests/CallVisualizer/ScreenSharing/Mocks/ScreenSharingView.Mock.swift

```diff
@@ -0,0 +1,26 @@
+#if DEBUG
+
+import UIKit
+
+extension CallVisualizer.ScreenSharingView.Props {
+    static func mock(
+        style: ScreenSharingViewStyle = .mock(),
+        header: Header.Props = .mock(),
+        endScreenSharing: ActionButton.Props = .mock()
+    ) -> CallVisualizer.ScreenSharingView.Props {
+        return .init(style: style, header: header, endScreenSharing: endScreenSharing)
+    }
+}
+
+#endif
+//endScreenSharing: .init(
```

Comment:
this can be deleted

Reply:
fixed [988c3dc](https://github.com/salemove/ios-sdk-widgets/pull/521/commits/988c3dc34073022cd6df7569db79d476a7a6646f)